src/util/performance.js

```diff
@@ -56,7 +56,7 @@ export const PerformanceUtils = {
     measure(name: string, begin?: string, end?: string) {
         performance.measure(name, begin, end);
     },
-    beginMeasure(name: string) {
+    beginMeasure(name: string): { mark: string, name: string } {
```

Comment:
Nit: now that this type is used in multiple places, let's cut it out and name it (e.g. `type PerformanceMark = {mark: string, name: string};`).

Reply:
I don't think this type is being used elsewhere? just tried searching the codebase

Reply:
It's accepted below in endMeasure, just has a different property order (which doesn't matter for types).